jujugui/static/gui/src/test/test_ui_state.js

```diff
@@ -1131,6 +1131,7 @@ describe('UI State object', function() {
     }];
 
     it('can generate proper urls from the state object', function() {
+      window.flags = {};
```

Comment:
I'm not sure this is a good idea - this means that as soon as we remove the react flag these tests will fail and we won't necessarily remember why.